cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,369 @@
+import pandas as pd
+import numpy as np
+from typing import Optional, cast, Dict, Any
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+from cleanlab.internal.util import get_num_classes
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Summarizes which classes in a multi-label dataset appear most often mislabeled overall.
+
+    This method works by providing any one (and only one) of the following inputs:
```

Comment:
```suggestion
    """Summarizes which classes in a multi-label dataset appear most often mislabeled overall.

    Since classes are not mutually exclusive in multi-label classification, this method summarizes the label issues for each class independently of the others.
    This method works by providing any one (and only one) of the following inputs:
```